cvat/settings/base.py

```diff
@@ -583,6 +583,10 @@ class CVAT_QUEUES(Enum):
         "LOCATION": f"redis://:{urllib.parse.quote(redis_ondisk_password)}@{redis_ondisk_host}:{redis_ondisk_port}",
         "TIMEOUT": CVAT_CHUNK_CACHE_TTL,
     },
+    "shared": {
+        "BACKEND": "django.core.cache.backends.redis.RedisCache",
```

Comment:
Can we just use Redis for the default cache? Is there any use case where a local memory cache is preferable?